### ISSUE TITLE
[SAP] [Accessibility] [Analytical Table]: selected component name (check box) is not conveyed

### ISSUE THREAD

**Issue by vraguna** [SAP/ui5-webcomponents-react#3776]
**Describe the bug**
UI type not recognized.

**To Reproduce**

Go to the corresponding [example in the docs](https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-analyticaltable--responsive-columns)
Open the developer tools 
Click on the "inspect element" icon 
Click on the "check box" on any of the analytical table rows

**Observed Behavior:** 
On accessing the checkbox in the rows, the user is not conveyed the UI type that it is a checkbox. 

Further analysis revealed that the “aria-label” dynamically change the text for selection and un-selection of the checkbox.

**Screenshots:**

![image](https://user-images.githubusercontent.com/42251879/202493695-ffd8dbc8-634a-44a4-9b0a-649188113c67.png)

![image](https://user-images.githubusercontent.com/42251879/202493187-6a44868e-ed38-4cf1-a92a-22f937c88e7a.png)

As you notice that there is no indication of the component (check box) that is being selected or un-selected.

Expected Behavior:  
All the UI elements should be presented correctly using assistive technologies. Here the user should be conveyed that the UI element accessed is a checkbox.

**Comment by Lukas742** [comment]
Hi @vraguna 

the checkbox is only there as a visual indicator, as you can see it cannot be focused as even if you click it the respective cell becomes the active element. Therefore I don't think we need to announce that it's a checkbox. 

For me the selected state is also announced (tested with NVDA and MacOS VoiceOver). It depends on how your `selectionBehavior` is set. With `Row` all cells will announce the text selection, with `RowSelector` only the selection cell.

I'm not really sure what you mean by that:
> Further analysis revealed that the “aria-label” dynamically change the text for selection and un-selection of the checkbox.

This is the intended behavior, so that the user knows that he can toggle the selection by pressing the spacebar.

**Comment by vraguna** [comment]
Hi @vraguna 

Thank you for the prompt response.

Yes, in our application, the selected state is announced upon clicking the particular cell (check box) since we have set the selectionBehavior to "RowSelector".

As you described above, it is clear now that the checkbox is just a visual indicator. Thanks for the clarification.
